src/main/riddl/Tenant/tenant.md

```diff
@@ -1,2 +1,4 @@
 The concept of Tenant in Improving.App is simply the finanically responsible entity for all things that 
-are created under that Tenant - Organizations, Members, Events, Venues, Locations, etc.
\ No newline at end of file
+are created under that Tenant - Organizations, Members, Events, Venues, Locations, etc. It is also the
+primary source of truth for all data in the app - every Organization, Member, Event, etc. must belong to 
```

Comment:
I don't think that tenant is the primary source of all data in the app. The Organization entity is the primary source of data for the Organization. Member for Member, and Event for Event. Organization must reference a valid Tenant, and Event must reference a valid Organization. But Member need not belong to a specific Tenant, Organization, or Event.